src/main/app/case/CaseApi.ts

```diff
@@ -28,12 +36,22 @@ export class CaseApi {
 
     const serviceCases = cases.filter(c => c.case_data.divorceOrDissolution === serviceType);
     switch (serviceCases.length) {
-      case 0:
+      case 0: {
         return false;
-      case 1:
-        return { id: serviceCases[0].id, state: serviceCases[0].state, ...fromApiFormat(serviceCases[0].case_data) };
-      default:
+      }
+      case 1: {
+        const { id, state, case_data: caseData } = serviceCases[0];
+        return {
+          ...fromApiFormat(caseData),
+          id,
+          state,
+          applicationFeeOrderSummary: caseData.applicationFeeOrderSummary,
```

Comment:
why are these two fields here? They're not special

Reply:
The main reason I put these here rather than in the `case.ts` was for security.

Anything in `case.ts` can be modified by the user via a specially crafted post request.